travis/tests/exponential/model/configuration/exponential.fac

```diff
@@ -0,0 +1,26 @@
+********************************************************************* ;
```

Comment:
should this maybe be called `firstorder.fac`, for consistency with the other test?